.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Typos and doc fixes
+about: Typos and doc fixes
+title: ''
+labels: documentation
+assignees: ''
+---
+
+## 📚 Documentation
+
+For typos and doc fixes, please go ahead and:
+
+1. Create an issue (You can skip this if it is a very simple fix).
+1. Fix the documentation.
+1. Make sure documentation builds without an error (See [Building the docs](https://pyodide.org/en/stable/development/contributing.html#building-the-docs)).
+1. Submit a PR.
+
+Note: If there is no change in code, we recommend you to append `[skip ci]` in your commit message.
```

Comment:
maybe,
```suggestion
<!-- Note: If there are no changes in code, we recommend you to append `[skip ci]` in your commit message to avoid starting the build CI.
-->
```

Reply:
Though it would be really great if we could figure out how to unconditionally run format checks. I guess last time I said this the status was "not clear that it is possible".

Reply:
Maybe one approach would be we could add the a manually triggered format checker to github actions and then run that on all `[skip ci]` PRs before merging?

Reply:
Hmm, well we can always go back to custom code that parses the commit message and kills CI it if required. That should be even easier in GH actions. Since manually triggered CI is also not ideal.

Reply:
Well we'd need it for both GH actions and CircleCI if we have both. For CircleCI, I guess the easy thing would be to make a new ancestor of all of the checks you don't want to run and fail it if the run isn't needed. But it would be nice to have "don't run dependent jobs but report as success".

Surely it's common for monorepos to have many independent subsystems and for people to only want to test the subsystem that changed? 

Anyways this discussion is a bit off topic here.